src/processRunner.ts

```diff
@@ -58,8 +57,6 @@ class CommandProcessExecution implements IProcessExecution {
                 if (!output) {
                     if (stdoutBuffer.length > 0) {
                         reject(new Error('Can not decode output from the process'));
-                    } else if (stderrBuffer.length > 0 && !this.commandProcess.killed) {
```

Comment:
In short, yeah.
The scenario was that if there is no testplan tests present (and testplan is enabled), it will of course try to discover the tests anyway. This is all expected.
However since testplan runs as a script, not as a module, like pytest, python will return the error
`python: can't open file 'test_plan.py': [Errno 2] No such file or directory`
to stderr, and with no stdout. Even if errno 2 is expected.
Since commandProcess.killed is already checked in the first "if", this "if" only means that "if there is no stdout, but there is stderr, reject even if error was expected"

Aaaand note to self, I removed the accepted errno 2 during the end refactor.